site/search/Autocomplete.tsx

```diff
@@ -264,6 +268,7 @@ export function Autocomplete({
                 return sources
             },
             plugins: [recentSearchesPlugin],
+            insights: getPreferenceValue(PreferenceType.Analytics),
```

Comment:
Hmmmm I think the fact that we're merging indexes might mess not be compatible with their default solution.

Because even if you click the "first" result that is a chart, it tracks it as position 4

If we want to fix this, I think we'd have to dig into how their library automatically works and try to reconstruct the event data and submitter ourselves.


https://github.com/owid/owid-grapher/assets/11844404/4d74d6c8-ead7-4435-a32a-f1b50065be90